narayana-bom-test/pom.xml

```diff
@@ -17,7 +17,7 @@
   <name>Narayana Bom Test</name>
   <description>Narayana BOM is for centralising dependency management for development purposes (test scoped dependencies)</description>
 
-<!--
+  <!--
```

Comment:
Also this one should go to a dedicated commit for style changes

Reply:
Strange, I don't recall making this change, I'll back it out.